plugins/omemo/src/ui/manage_key_dialog.vala

```diff
@@ -58,7 +58,7 @@ public class ManageKeyDialog : Gtk.Dialog {
         verify_yes_button.clicked.connect(() => {
             confirm_image.set_from_icon_name("security-high-symbolic", IconSize.DIALOG);
             confirm_title_label.label = _("Verify key");
-            confirm_desc_label.set_markup(_("Once confirmed, any future messages sent by %s using this key will be highlighted accordingly in the chat window.").printf(@"<b>$(device[db.identity_meta.address_name])</b>"));
+            confirm_desc_label.set_markup(_("Future messages sent by %s and encrypted with this key will be highlighted accordingly in the chat window.").printf(@"<b>$(device[db.identity_meta.address_name])</b>"));
```

Comment:
Right, so would "Future encrypted messages sent by %s and signed with this key..." be fine?

I'd actually use an expression like "sent by %s from an app using this key". I remember that you wanted to refer to devices instead of apps, though, so I hope the proposition above is a good compromise.

Reply:
> I'd actually use an expression like "sent by %s from an app using this key"

We could perhaps do

> Future messages sent by %s from a device that uses this key will be highlighted accordingly in the chat window.